R/table.R

```diff
@@ -63,31 +63,49 @@ html_table.xml_node <- function(x, header = NA, trim = TRUE,
 
   ncols <- lapply(cells, html_attr, "colspan", default = "1")
   ncols <- lapply(ncols, as.integer)
+  nrows <- lapply(cells, html_attr, "rowspan", default = "1")
+  nrows <- lapply(nrows, as.integer)
```

Comment:
Here nrows, like ncols, is a list of vectors so I don't think this would work. And you need this structure (i.e. list of vectors) for filling later on.